sql/sqlite/SQLite.g4

```diff
@@ -832,6 +881,25 @@ K_WHEN : W H E N;
 K_WHERE : W H E R E;
 K_WITH : W I T H;
 K_WITHOUT : W I T H O U T;
+K_FIRST_VALUE : [Ff][Ii][Rr][Ss][Tt][_][Vv][Aa][Ll][Uu][Ee];
+K_OVER  : O V E R;
+K_PARTITION : P A R T I T I O N;
+K_RANGE     : R A N G E;
+K_PRECEDING : P R E C E D I N G;
+K_UBOUNDED   : U N B O U N D E D;
+K_CURRENT   : C U R R E N T;
+K_FOLLOWING : F O L L O W I N G;
+K_CUMEDIST  : [Cc][Uu][Mm][Ee][_][Dd][Ii][Ss][Tt];
+K_DENSERANK : [Dd][Ee][Nn][Ss][Ee][_][Rr][Aa][Nn][Kk];
+K_LAG       : L A G;
+K_LASTVALUE  : [Ll][Aa][Ss][Tt][_][Vv][Aa][Ll][Uu][Ee];
+K_LEAD       : L E A D;
+K_NTHVALUE   : [Nn][Tt][Hh][_][Vv][Aa][Ll][Uu][Ee];
+K_NTILE      : N T I L E;
+K_PERCENTRANK : [Pp][Ee][Rr][Cc][Ee][Nn][Tt][_][Rr][Aa][Nn][Kk];
+K_RANK       : R A N K;  
+K_ROWNUMBER  : [Rr][Oo][Ww][_][Nn][Uu][Mm][Bb][Ee][Rr];
```

Comment:
Should these new keywords not also be part of the `keywords` production?

Reply:
Already checked ,There is no duplicacy of Tokens